packages/admin/admin/src/dataGrid/useDataGridRemote.tsx

```diff
@@ -37,7 +37,9 @@ export function useDataGridRemote({
         (!parsedSearch.sort
             ? undefined
             : !Array.isArray(parsedSearch[sortParamName])
-            ? [parsedSearch[sortParamName] as string]
+            ? parsedSearch[sortParamName] === "[]"
```

Comment:
Should we change it to `none` instead? I thought that an empty array works out of the box with the query-string package, but it seems that it doesn't

Reply:
i tried it with an empty array but doesn't work. Yes we can use `none` instead. @nsams whats your opinion?

Reply:
Examples:
ASC: `?sort=title%3Aasc`
DESC: `?sort=title%3Adesc`
No Sorting: `?sort=%5B%5D`
2 Spalten ASC: `sort=description%3Aasc&sort=title%3Aasc`
1 ASC, 1 DESC: `sort=title%3Aasc&sort=price%3Adesc`
for 3 columns: `sort=title%3Aasc&sort=price%3Adesc&sort=type%3Aasc`

if you have more than 1 column and you set 1 column to "no sorting" the column disapears in the query-string.

Reply:
given those examples `[]` is indeed odd and `none` as magic string makes more sense

Reply:
done